mysqlclient-src/build.rs

```diff
@@ -1,14 +1,19 @@
 fn main() {
-    let openssl_dir = std::env::var("DEP_OPENSSL_ROOT").unwrap();
+    // `DEP_OPENSSL_ROOT` is only set if openssl-src is used
+    let openssl_dir = std::env::var("DEP_OPENSSL_ROOT");
```

Comment:
Given that this environment variable is only conditionally set if `openssl-src` is used I wonder whether we even need a new feature for this at all or just could detect if this variable is set or not and rely on the user to specify how they want to build openssl by enabling the relevant features for the openssl crate?

Reply:
Ill have to check this again. And see if i can mimic `pq-sys` / `pq-src` a bit more for this repo. That would be a better solution then i think.